app/Http/Controllers/User/ProfileController.php

```diff
@@ -52,34 +64,80 @@ public function show($id)
     /**
      * Show the form for editing the specified resource.
      *
-     * @param  int  $id
+     * @param  int $id
      * @return \Illuminate\Http\Response
      */
     public function edit($id)
     {
-        //
+        $user = User::findOrFail($id);
+
+        return response()->json([
+            'error' => false,
+            'user' => $user,
+        ]);
     }
 
     /**
      * Update the specified resource in storage.
      *
-     * @param  \Illuminate\Http\Request  $request
-     * @param  int  $id
+     * @param  \Illuminate\Http\Request $request
+     * @param  int $id
      * @return \Illuminate\Http\Response
      */
     public function update(Request $request, $id)
     {
-        //
+        $data = $request->all();
+
+        DB::beginTransaction();
+
+        User::where('id', '=', $id)->update([
+            'name' => $request->name,
+            'nick_name' => $request->nick_name,
+            'phone' => $request->phone,
+            'address' => $request->address,
+            'gender' => $request->gender,
+        ]);
+
+        DB::commit();
+
+        return response()->json([
+            'error'     =>  false,
+            'message'   =>  trans('message.success'),
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.SpaceAfterDoubleArrow</br>**Severity**: error</br>**File**: app/Http/Controllers/User/ProfileController.php</br>**Line**: 105</br>Expected 1 space between double arrow and "trans"; 2 found